cypress/tests/chat.cy.ts

```diff
@@ -424,4 +396,57 @@ describe('chat', () => {
     cy.getByText('Estetyt keskustelut', 'a');
     cy.get('div[aria-label="unseen-messages-dot"]').should('not.exist');
   });
+
+  it('can report mentor succesfully', () => {
```

Comment:
I would move these report -tests to "report.cy.ts"

Reply:
Yeah good suggestion!

Reply:
Done!